test/integration_test/main_test.go

```diff
@@ -51,6 +51,26 @@ func setup() error {
 		"upgrade-hops-url-list",
 		"",
 		"List of Portworx Spec Generator URLs separated by commas used for upgrade hops")
+	flag.StringVar(&ci_utils.CloudProvider,
+		"cloud-provider",
+		"",
+		"Type of cloud provider")
+	flag.StringVar(&ci_utils.PxEnvVars,
+		"portworx-env-vars",
+		"",
+		"List of comma separated environment variables that will be added to StorageCluster spec")
+	flag.StringVar(&ci_utils.PxVolumes,
```

Comment:
Looks like this is being used only for cloud storage. Can we rename it to something like `PxDeviceSpecs`?